LICENSE

```diff
@@ -1,6 +1,19 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Justin Ellis
+*** Copyright Notice ***
+PTMCMCSampler Copyright (c) 2015 to 2021, Justin Ellis
+All rights reserved.
+
+PTMCMCSampler Copyright (c) 2022, Mark Forrer on behalf of
+National Technology & Engineering Solutions of Sandia, LLC.
+All rights reserved.
+
+NOTICE.  This Software was partially developed under funding from the U.S. Department
```

Comment:
I don't like having this here and I feel like there have to be other open source packages with DoE contributors that don't have this disclaimer. MIT is very open so I don't really want to have this in here.

Reply:
Thanks for your feedback, and I'll communicate this to my management.  For what it's worth, I advocated against any changes to this file, and I'm not fond of having to be an intermediary in this conversation.  For the moment, pending additional feedback, my hands are tied.

Reply:
I had a conversation with my management about this, and they agreed with your feedback that the MIT license is sufficient to handle the license statement proposed in this version.  It's now pared down to just an acknowledgment of the DoE having funded part of the work.  I hope that's more acceptable, and assume it may make sense to also acknowledge other contributions as needed (though maybe following this PR).

Reply:
Sorry for the trouble hear and thank you for working it out